packages/beacon-node/src/chain/produceBlock/validateBlobsAndKzgCommitments.ts

```diff
@@ -1,61 +1,54 @@
-import {CELLS_PER_EXT_BLOB, ForkName, ForkSeq, isForkPostDeneb} from "@lodestar/params";
-import {ExecutionPayload, fulu} from "@lodestar/types";
-import {BlobsBundle} from "../../execution/index.js";
+import {CELLS_PER_EXT_BLOB} from "@lodestar/params";
+import {deneb, fulu} from "@lodestar/types";
 import {kzg} from "../../util/kzg.js";
 
 /**
  * Optionally sanity-check that the KZG commitments match the versioned hashes in the transactions
  * https://github.com/ethereum/consensus-specs/blob/11a037fd9227e29ee809c9397b09f8cc3383a8c0/specs/eip4844/validator.md#blob-kzg-commitments
  */
-
 export async function validateBlobsAndKzgCommitments(
-  fork: ForkName,
-  _payload: ExecutionPayload,
-  blobsBundle: BlobsBundle,
-  cells?: fulu.Cell[][]
+  commitments: deneb.KZGCommitment[],
+  proofs: deneb.KZGProof[],
+  blobs: deneb.Blobs
 ): Promise<void> {
-  if (!isForkPostDeneb(fork)) {
-    throw Error(`validateBlobsAndKzgCommitments called with pre-deneb fork=${fork}`);
+  if (blobs.length !== commitments.length) {
+    throw Error(`Blobs bundle blobs len ${blobs.length} != commitments len ${commitments.length}`);
   }
 
-  if (blobsBundle.blobs.length !== blobsBundle.commitments.length) {
-    throw Error(
-      `Blobs bundle blobs len ${blobsBundle.blobs.length} != commitments len ${blobsBundle.commitments.length}`
-    );
+  if (proofs.length !== blobs.length) {
+    throw new Error(`Invalid proofs length for BlobsBundleV1 format: expected ${blobs.length}, got ${proofs.length}`);
   }
 
-  if (ForkSeq[fork] < ForkSeq.fulu) {
-    if (blobsBundle.proofs.length !== blobsBundle.blobs.length) {
-      throw new Error(
-        `Invalid proofs length for BlobsBundleV1 format: expected ${blobsBundle.blobs.length}, got ${blobsBundle.proofs.length}`
-      );
-    }
-
-    if (!(await kzg.asyncVerifyBlobKzgProofBatch(blobsBundle.blobs, blobsBundle.commitments, blobsBundle.proofs))) {
-      throw new Error("Error in verifyBlobKzgProofBatch");
-    }
-
-    return;
+  if (!(await kzg.asyncVerifyBlobKzgProofBatch(blobs, commitments, proofs))) {
+    throw new Error("Error in verifyBlobKzgProofBatch");
   }
+}
 
-  if (!cells) {
-    throw Error(`Missing cells for post-fulu fork=${fork}`);
+/**
+ * Optionally sanity-check that the KZG commitments match the versioned hashes in the transactions
+ */
+export async function validateCellsAndKzgCommitments(
+  commitments: deneb.KZGCommitment[],
+  proofs: deneb.KZGProof[],
```

Comment:
these should be cell proofs?

```suggestion
  proofs: fulu.KZGProof[],
```